compendium/modules/w04-objects-lab.tex

```diff
@@ -66,7 +66,7 @@ \subsection{Obligatoriska uppgifter}
 Kompilera ditt program med \texttt{scala-cli compile .} och kontrollera med \\\texttt{ls .scala-build/*/classes/*} att några filer som slutar på \texttt{class} har skapats i en underkatalog. \Pen Vilket namn har underkatalogen med ditt programs maskinkodsfiler? Varför fick underkatalogen detta namn?
 
 \Subtask
-Kör kommandot \texttt{scala-cli run . --main-class blockmole.run} för att exekvera ditt program och kontrollera utskriften i terminalfönstret.
+Kör kommandot \texttt{scala run . ---main-class blockmole.run} för att exekvera ditt program och kontrollera utskriften i terminalfönstret.
```

Comment:
oops merged before I saw this: `\texttt` should be shanged to `\code` top get correct dashes. new PR welcome

Reply:
Fixed #818